speccy/machine.go

```diff
@@ -16,14 +19,10 @@ type Machine struct {
 }
 
 func NewMachine(z *zog.Zog) *Machine {
-	screen, err := NewScreen(z.Mem)
-	if err != nil {
-		panic(fmt.Sprintf("Can't create screen: %s", err))
-	}
 
 	return &Machine{
 		keys:   NewKeyboardState(),
-		screen: screen,
+		screen: nil,
```

Comment:
Idiomatic golang doesn't specify zero initialisers (however uncomfortable it may make C programmers)